studio/schemas/documents/siteSettings/locale.ts

```diff
@@ -0,0 +1,41 @@
+import { defineField, defineType } from "sanity";
+
+import { relevantCurrencies } from "studio/utils/currencies";
+import { relevantLocales } from "studio/utils/locales";
+
+export const localeID = "locale";
+
+const locale = defineType({
+  name: localeID,
+  type: "document",
+  fields: [
+    defineField({
+      name: "locale",
+      type: "string",
+      title: "Region",
```

Comment:
I think I would name this currency and not region

Reply:
Currency is the field below. This is technically "Locale", but "Region" is less techy.

Reply:
I was a bit fast commenting this! The level above could be renamed, but region is fine here